dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -0,0 +1,343 @@
+		const europeModal = document.getElementById('europe-modal-dialog');
+		if (
+			europeModal instanceof HTMLDialogElement &&
+			modalType !== 'NoModal'
+		) {
+			europeModal.showModal();
+			europeModal.addEventListener('close', () => {
+				hideModal();
+			});
+			document.documentElement.style.overflow = 'hidden';
+		}
+	}, []);
+
+	const confirmNewEdition = (editionId: EditionId) => {
+		setCookie({
+			name: 'GU_EDITION',
+			value: editionId,
+			isCrossSubdomain: true,
+		});
+		setCookie({
+			name: 'GU_eu_modal_dismissed',
+			value: 'true',
+			daysToLive: 100, //todo - check how long
+		});
+		if (editionId === edition) {
+			hideModal();
+		} else {
+			window.location.replace(getEditionFromId(editionId).url);
+		}
+	};
+
+	const dismissModal = () => {
+		setCookie({
+			name: 'GU_eu_modal_dismissed',
+			value: 'true',
+			daysToLive: 100, //todo - check how long
+		});
+		hideModal();
+	};
+
+	const hideModal = () => {
+		const europeModal = document.getElementById('europe-modal-dialog');
+		if (europeModal instanceof HTMLDialogElement) {
+			europeModal.close();
+			document.documentElement.style.overflow = '';
+		}
+	};
+
+	return (
+		<dialog css={dialogStyles} id={'europe-modal-dialog'}>
+			<div css={styles}>
+				{!switchEdition ? (
+					<>
+						<div css={textStyles}>
+							<h1 css={headlineStyles}>
+								{modalType === 'ModalSwitched' &&
+									'We’ve switched you to the new Europe edition of the Guardian'}
+								{modalType === 'ModalDoYouWantToSwitch' &&
+									'Would you like to switch to our new Europe edition?'}
+							</h1>
+							<p css={bodyStyles}>
+								{modalType === 'ModalSwitched' &&
+									'You’re now getting more coverage tailored for readers in Europe'}
+								{modalType === 'ModalDoYouWantToSwitch' &&
+									'We’ve launched a new edition of the Guardian with our global coverage tailored for readers in Europe'}
+							</p>
+							<div css={buttonDivStyles}>
+								{modalType === 'ModalSwitched' && (
+									<Button
+										size={'small'}
+										onClick={() => {
+											confirmNewEdition('EUR');
+										}}
+										cssOverrides={OKButtonStyles}
+									>
+										OK, Thanks
+									</Button>
+								)}
+								{modalType === 'ModalDoYouWantToSwitch' && (
+									<Button
+										size={'small'}
+										onClick={() => dismissModal()}
+										cssOverrides={OKButtonStyles}
+									>
+										{' '}
+										No, Thanks{' '}
+									</Button>
+								)}
+								<Button
+									size={'small'}
+									priority={'subdued'}
+									onClick={() => setSwitchEdition(true)}
+								>
+									{' '}
+									Switch Edition{' '}
+								</Button>
+							</div>
+						</div>
+						<div css={imageStyles}>
+							<SvgFlagsInCircle />
+						</div>
+					</>
+				) : (
+					<div css={editionSectionDivStyles}>
+						<RadioGroup
+							error=""
+							label="Choose your edition"
+							name="colours"
+							orientation="vertical"
```

Comment:
```suggestion
						<RadioGroup
							label="Choose your edition"
							name="colours"
```

The default orientation is vertical and we're not using the error property! We might also be able to remove "name" too, but that one might be non optional!